### PR TITLE
Clean list

### LINKED ISSUE #89: CLEANUP: Style List View
Style the list view so it looks amazing
Also friends list view

### PR DESCRIPTION
**Pull request by asu91**
- added and styled header for Pin List and Friend List
- styled Pin List Item
- added Facebook pics to Friend List Item

closes #89

### CODE REVIEW

File: app/components/PinListItem.js [hunk 1]
```diff
@@ -124,29 +140,47 @@ export default class PinListItem extends Component {
...
-  undertext: {
-    flexDirection: 'row',
+  distance: {
+    fontSize: 19,
+    fontStyle: 'italic',
+  },
+  pin: {
+    flex: 1,
     alignSelf: 'center',
+    height: 45,
```
Comment (jake-shasteen) [inline app/components/PinListItem.js]: Change `height` to `50` to avoid cut-off at top of pin

File: app/components/PinListItem.js [hunk 2]
```diff
@@ -82,7 +92,7 @@ export default class PinListItem extends Component {
 
   render() {
     const { pin, targetPin, currLoc } = this.props;
-    let name = '';
+    let name = 'Your Pin';
```
Comment (jake-shasteen) [inline app/components/PinListItem.js]: Nice change

**Comment by jake-shasteen** [comment]
Mergable :heavy_check_mark:

**Comment by bignell** [comment]
@asu91 @jake-shasteen Looks good.

**Pull request merged**